src/device-registry/controllers/grid.controller.js

```
@@ -807,369 +242,95 @@ const createGrid = {
       const result = await gridUtil.createGridFromShapefile(request, next);
-      if (isEmpty(result) || res.headersSent) {
-        return;
-      }
-      if (result.success === false) {
-        const status = result.status
-          ? result.status
-          : httpStatus.INTERNAL_SERVER_ERROR;
-        return res.status(status).json(result);
-      }
+      handleResponse({ res, result });
     } catch (error) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)